src/components/SEOTags.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import { NextSeo } from 'next-seo'
+
+export default function SEOTags({ pageTitle, pageDescription, pageUrl }) {
+  return (
+    <NextSeo
+      title={pageTitle}
+      description={pageDescription}
+      openGraph={{
+        url: `http://${pageUrl}`,
```

Comment:
It looks like in a final snippet there will be `http://my.optrispace.com` instead of `https://my.optrispace.com/jobs/${job.id}`. Isn't it?

Reply:
Shouldn't it be better to add active page's url?

Reply:
Frankly speaking, if I see a link in social media, I would expect to get into the specific job page, instead of homepage of the platform.